### ISSUE TITLE
shouldn't loss have add(-1) here?

### ISSUE THREAD

**Issue by mirachakshu** [SherlockLiao/pytorch-beginner#42]
https://github.com/L1aoXingyu/pytorch-beginner/blob/61db1de8a2528ab50cd64b50af2268a8b3bc01a4/08-AutoEncoder/Variational_autoencoder.py#L87

**Comment by TakshDhabalia** [comment]
Yep, was wondering the same thing but there is surely a better explanation to that because it is correct
